codespell_lib/data/dictionary.txt

```diff
@@ -32382,6 +32382,8 @@ prepere->prepare
 prepered->prepared
 preperes->prepares
 prepering->preparing
+preperint->preprint
+preperints->preprints
```

Comment:
Perhaps add all verbal forms?
```
preperinted->preprinted
preperinting->preprinting
```

Reply:
why not?
```suggestion
preperints->preprints
preperinted->preprinted
preperinting->preprinting
```

Reply:
ha -- I hit some github bug which forbids me to commit this suggestion... try or I will try later

Reply:
Actually, verbals forms are very uncommon: couldn't find the typo in GitHub or with Google.